### ISSUE TITLE
Possibility to disable VNC for Engines while still having remote front panel display 

### ISSUE THREAD

**Issue by hlnorris3** [zynthian/zynthian-issue-tracking#843]
**Is your feature request related to a problem? Please describe.**
For certain engines, VNC consumes a lot of CPU. However still having access to the front panel remotely is useful when the Zynthian is located remotely.

**Describe the solution you'd like**
Would like to have the option to disable VNC from just the engines. OK to disable for all engines.

**Describe alternatives you've considered**
Others have reported that enabling the fake GL drivers reduces the VNC CPU load for Zynaddsubfx (one of the heavier VNC hogs) and trying it, it seemed to do so but "broke" the Zynthian native display. Approach not pursued further. 

**Additional context**

**Comment by riban-bw** [comment]
I agree that it is advantageous to allow each VNC server to be enabled individually. A user may want / need to use VNC to access the main UI but not want the overhead of naitive plugin GUIs. Similarly a user may want to access the naitive GUIs but not want the overhead of the main UI. (Although the latter has less impact on resources.)

I would suggest adding seperate options in admin / webconf for each VNC service. @jofemodo do you concur?
